src/pages/Liquidated.tsx

```diff
@@ -8,41 +8,28 @@ import { LiquidatedVaultCountCard } from '@/widgets/LiquidatedVaultCountCard';
 import { VaultStatesChart } from '@/widgets/VaultStatesChart';
 import { populateMissingDays, subQueryFetcher } from '@/utils';
 import { LIQUIDATIONS_DASHBOARD, LIQUIDATION_GRAPH_TOKENS_QUERY, LIQUIDATION_DAILY_METRICS_QUERY } from '@/queries';
-import { GraphData, LiquidationDashboardResponse, LiquidationMetricsDailyResponse, OraclePriceNode, TokenNamesResponse, VaultManagerGovernancesNode } from '@/types/liquidation-types';
-import { GRAPH_DAYS } from '@/constants';
-
+import {
+  GraphData,
+  LiquidationDashboardResponse,
+  LiquidationMetricsDailyResponse,
+  TokenNamesResponse,
+} from '@/types/liquidation-types';
+import { GRAPH_DAYS, SUBQUERY_STAGING_URL } from '@/constants';
 
 const sum = (items: Array<string>) => items.reduce((agg, next) => agg + Number(next), 0);
 
 export function Liquidated() {
-  const { data, isLoading } = useSWR<AxiosResponse, AxiosError>(LIQUIDATIONS_DASHBOARD, subQueryFetcher);
-  const response: LiquidationDashboardResponse = data?.data?.data;
-
-  const vaultManagerGovernances: { [key: string]: VaultManagerGovernancesNode } =
-    response?.vaultManagerGovernances?.nodes?.reduce((agg, node) => {
-      const idSegments = node?.id?.split('.');
-      if (idSegments.length < 4) {
-        throw new Error(`Node ID does not contain enough segments: ${node.id}`);
-      }
-      const managerName = idSegments.slice(0, 4).join('.');
-      return { ...agg, [managerName]: node };
-    }, {});
-  const oraclePrices: { [key: string]: OraclePriceNode } = response?.oraclePrices?.nodes?.reduce(
-    (agg, node) => ({ ...agg, [node.typeInName]: node }),
-    {},
+  const { data, isLoading } = useSWR<AxiosResponse, AxiosError>(LIQUIDATIONS_DASHBOARD, (query: string) =>
+    subQueryFetcher(query, SUBQUERY_STAGING_URL),
   );
+  const response: LiquidationDashboardResponse = data?.data?.data;
 
   const liquidationDashboardData = {
-    vaultManagerGovernances,
     vaultLiquidations: response?.vaultLiquidations?.nodes,
-    oraclePrices,
   };
 
   //  Queries for graph
-  const { data: tokenNamesData } = useSWR<AxiosResponse, AxiosError>(
-    LIQUIDATION_GRAPH_TOKENS_QUERY,
-    subQueryFetcher,
-  );
+  const { data: tokenNamesData } = useSWR<AxiosResponse, AxiosError>(LIQUIDATION_GRAPH_TOKENS_QUERY, subQueryFetcher);
```

Comment:
Can you please add error handling for the data you're fetching for `LIQUIDATION_GRAPH_TOKENS_QUERY` and `SUBQUERY_STAGING_URL`

Reply:
can you specifiy what you mean by error handling? this is not a new api call, but instead a modification of a previous api call which didnt have any error handling

Reply:
Handle cases when `useSwr` is loading the data or if some error occurs.